CONTRIBUTING.md

```diff
@@ -0,0 +1,116 @@
+
+```
+git clone url-you-just-copied
+```
+
+where `url-you-just-copied` is the url to this repository (your fork of this project).
+
+### Creating new branch to start dealing with the issue
+
+Change to the FCC forum clone directory on your computer (if you are not already there):
+
+```
+cd fcc-forum-clone
+```
+
+Now create a branch and switch to that new branch using the `git checkout` command. _One way to name a branch would be: `<number-of-issue>-<type-of-changes>-<what-the-issue-is-about>`_
+
+```
+git checkout -b your-new-branch-name
+```
+
+### Commit changes and pushing them
+
+Commit the changes once you are happy with them.
+If you go to the project directory and execute the command `git status`, you'll see there are changes.
+Add those changes to the branch you just created using the `git add` command:
+
+```
+git add changed-file-name
+```
+
+Now commit those changes to your local branch using the `git commit` command. _Try to make commit message descriptive, but not super verbose!_
+
+```
+git commit -m "commit message"
+```
+
+Push your local branch and create a remote branch on GitHub using the command `git push` :
+
+```
+git push -u origin branch-name
+```
+
+### Pull Request
+
+When you're finished with the changes, create a pull request, also known as a PR.
+
+- Please follow all instructions in the template so that we can review your PR.
+- Don't forget to [link PR to issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) if you are solving one.
+- Enable the checkbox to [allow maintainer edits](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) so the branch can be updated for a merge.
+  Once you submit your PR, a team member will review your proposal. We may ask questions or request for additional information.
+- We may ask for changes to be made before a PR can be merged, either using [suggested changes](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request) or pull request comments. You can make any other changes in your fork, then commit them to your branch.
+- As you update your PR and apply changes, mark each conversation as [resolved](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#resolving-conversations).
+- If you run into any merge issues, checkout this [git tutorial](https://github.com/skills/resolve-merge-conflicts) to help you resolve merge conflicts and other issues.
+
+### Still a little bit lost?
+
+- Check out [this video](https://www.youtube.com/watch?v=RGOj5yH7evk) from freeCodeCamp youTube channel Git and GitHub basics.
+- Don't be shy to ask us about anything related to the project and working on it! Depending on your question, use pull request threads, issue threads or the discussions!
```

Comment:
I don't think we need either exclamation point here. Periods would work best here.